CONTRIBUTING.md

```diff
@@ -155,7 +167,7 @@ systemProp.test.couch.http=[http|https] # default 5984
 systemProp.test.couch.ignore.compaction=[true|false] # default false
 systemProp.test.couch.auth.headers=[true|false] # default false
 ```
-Note: some tests need to be ignored based on the configuration of the CouchDB instance you are using or if you are running against Cloudant. For example if the compaction endpoint is unavailable eg returns a 403 response, these tests should be disabled. 
+Note: some tests need to be ignored based on the configuration of the CouchDB instance you are using or if you are running against Cloudant. For example if the compaction endpoint is unavailable eg returns a 403 response, these tests should be disabled.
```

Comment:
Interesting, I'd use full stops for e.g. and i.e., but not etc. For ease, let's use "such as returning a 403 response code"?